### PR TITLE
App: Fix dialogs and form uniformity

### PR DESCRIPTION
**Pull request by kyriediculous**
- Fix modal/dialog sizes to be uniform
- Fix button lengths to fill container
- In exit pool tokens show LP tokens to receive in selector component instead of a separate component
- Add token icons where we didn't have them
- Use text input fields instead of number for swap (like we do elsewhere)

### CODE REVIEW

File: packages/app/src/components/swap/ConfirmSwapModal.tsx [hunk 1]
```diff
@@ -118,18 +130,25 @@ const ConfirmSwapModal: FC<Props> = ({
...
+              <Box pad={{ top: "medium", horizontal: "large" }} align="center">
+                {confirmStatus === "None" && (
+                  <Form style={{ width: "100%" }}>
+                    <Box gap="medium">
                       <FormField label={`Send ${tokenSendedSymbol}`}>
                         <TextInput
                           readOnly
                           id="formSwapSend"
-                          type="number"
+                          type="text"
```
Comment (vujevits) [inline packages/app/src/components/swap/ConfirmSwapModal.tsx:142]: if you did that to disable the up and down arrows in firefox, there's a better way of doing it:
https://stackoverflow.com/questions/23372903/hide-up-down-arrow-buttons-spinner-in-input-number-firefox-29
I think we are already doing thix in index.css but with a webkit prefix, maybe that's the issue

Comment (kyriediculous) [reply packages/app/src/components/swap/ConfirmSwapModal.tsx:142]: We are using text input fields everywhere else. Would be good to stick to one for consistency. The actual type the state begins with is also a string.

Comment (vujevits) [reply packages/app/src/components/swap/ConfirmSwapModal.tsx:142]: it's not a question of type but what the form allows to input. you cannot type text to numeric inputs - it's a string nonetheless

**Comment by vujevits** [comment]
the `join` dialog does not fit on my 15inch screen if both the approval button and the validation warning is present (a scrollbar appears inside the dialog)

**Comment by kyriediculous** [comment]
@vujevits care to review before I rebase ?

**Review by vujevits — DISMISSED** [review]
looks good ⚡

**Comment by kyriediculous** [comment]
rebased, waiting for build to complete and do some sanity checks before merging

File: packages/app/src/components/actions/deposit.tsx [hunk 2]
```diff
@@ -90,13 +95,13 @@ const Deposit: FC<Props> = ({ name, symbol, logo, tokenBalance, tenderTokenBalan
               <FormField
                 fill
                 label="Deposit Amount"
-                name="depositAmount"
+                protocolName="depositAmount"
```
Comment (vujevits) [inline packages/app/src/components/actions/deposit.tsx:98]: of course I did not catch this before earlier, this broke input validation @kyriediculous

**Pull request merged**